debian/rules

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --builddirectory=_build
+
+override_dh_auto_configure:
+	dh_auto_configure -- \
+	--prefix=${DEB_PREFIX} \
```

Comment:
I found two issues which I haven't found other ways to solve: a) make it use multi-arch path for libdir and b) dh_auto_configure receives paths without substitution:

```
./configure --build=x86_64-linux-gnu --prefix=/usr --includedir=\${prefix}/include --mandir=\${prefix}/share/man --infodir=\${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --disable-silent-rules --libexecdir=\${prefix}/lib/emeus --disable-maintainer-mode --disable-dependency-tracking
```

I don't know really if we give special treatment to those in our configure file.